lasy-fuzzy-syntax.md

```diff
@@ -0,0 +1,25 @@
+## motivation
+Описывает набор инструкций, которые будут выполняться только тогда, когда внешнему коду потребуется результат исполнения этих инструкций.
+
```

Comment:
It describes a set of instructions that will be executed only when the external code needs the result of executing these instructions.